spec/deaf_grandma_spec.rb

```diff
@@ -9,10 +9,9 @@
   end
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
-  end
-
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
+  end 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    expect{ 3.times {script.speak("BYE")} }.to raise_error(SystemExit, "SEE YOU LATER SONNY!")
```

Comment:
Should this raise a `SystemExit` error? Should your program have its own error type or use `RuntimeError`?